doc/explanations/testing-and-debugging.md

```diff
@@ -0,0 +1,76 @@
+Robotoff is an API that pulls prediction data, annotation data, product data, nutrition data from MongoDB database in Open Food Facts server.
```

Comment:
I could remove this introduction if required.

But if the user does not understand this they wouldn't understand why they should use Factory or write test cases when Robotoff connects to the OFF server.